opal/runtime/opal_info_support.c

```diff
@@ -197,9 +200,30 @@ int opal_info_init(int argc, char **argv, opal_cmd_line_t *opal_info_cmd_line)
     if (!cmd_error
         && (opal_cmd_line_is_taken(opal_info_cmd_line, "help")
             || opal_cmd_line_is_taken(opal_info_cmd_line, "h"))) {
+        want_help = true;
+    }
+
+    /* Determine color support */
+    if(!cmd_error && !want_help) {
+        char *color = (opal_cmd_line_is_taken(opal_info_cmd_line, "color") ?
+            opal_cmd_line_get_param(opal_info_cmd_line, "color", 0, 0) : "auto");
+
+        if (0 == strcmp(color, "auto")) {
+            
```

Comment:
why blank line?  I'm also not sure I love leaving opal_info_color unset here.

Reply:
This is where the ansi/tty check will be implemented, which to-do (the emptyness is misleading indeed, sorry). Generally, auto means colors enabled, unless not supported (no ansi support, pipe etc, not entirely sure how this will be done and if these will be distinct paths)

Reply:
Check out this:

https://github.com/open-mpi/ompi/blob/ee324be152c1846161dc0553509a0ea4e4534d32/opal/runtime/opal_info_support.c#L873-L889